include/twin_private.h

```diff
@@ -15,18 +15,30 @@
 #define maybe_unused __attribute__((unused))
 
 /*
- * Post-transformed points are stored in 12.4 fixed point
- * values
+ * Fixed-point type definitions
+ *
+ * “x.y” is a representation of a fixed-point type,
```

Comment:
Avoid using `x.y` notation, which may confuse with the coordinates.

Reply:
Replace `x.y` with `m.f`